src/SQLChecks/Functions/Public/Get-TempDBConfiguration.ps1

```diff
@@ -0,0 +1,29 @@
+Function Get-TempDBConfiguration {
+    [cmdletbinding()]
+    Param(
+        [Parameter(ParameterSetName = "Config", ValueFromPipeline = $true, Position = 0)]
+        $Config
+
+        , [Parameter(ParameterSetName = "Values")]
+        $ServerInstance
+   
+    )
+
+    if ($PSCmdlet.ParameterSetName -eq "Config") {
+        $ServerInstance = $Config.ServerInstance
+    }
+
+    $query = @"
+    select  count(*) NumberOfFiles
+       ,sum(size / 128) [TotalSizeMB]
+    from    tempdb.dbo.sysfiles
```

Comment:
This is a SQL2000 compatibility view - should use `sys.database_files` going forwards.

Reply:
stay cognisant of the subtle differences between `database_files` & `master_files` in an AG setup as well

https://dba.stackexchange.com/a/220402/68127

Reply:
will update on the next iteration